### PR TITLE
TM-4707: Feature/admin job categories

### PR DESCRIPTION
**Pull request by pveland621**
This PR covers the functionality of loading in data for the job categories dropdown and also loading all skills and checking the box if the skill is associated with the selected job category.

Follow up PR will address the ability to save edits to the skills associated with a job category

Dual Merge: 
- [API PR](linkToAPIPR)

[Ticket](https://metaphase.atlassian.net/browse/TM-4707)

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -10348,10 +10348,10 @@ node-libs-browser@^2.2.1:
     util "^0.11.0"
     vm-browserify "^1.0.1"
 
-node-mocks-http@^1.11.0:
-  version "1.11.0"
-  resolved "https://registry.yarnpkg.com/node-mocks-http/-/node-mocks-http-1.11.0.tgz#defc0febf6b935f08245397d47534a8de592996e"
-  integrity sha512-jS/WzSOcKbOeGrcgKbenZeNhxUNnP36Yw11+hL4TTxQXErGfqYZ+MaYNNvhaTiGIJlzNSqgQkk9j8dSu1YWSuw==
+node-mocks-http@1.12.2:
+  version "1.12.2"
+  resolved "https://registry.yarnpkg.com/node-mocks-http/-/node-mocks-http-1.12.2.tgz#382b1824f625c2d41da82efedde936b9404e7fdd"
```
Comment (scott062) [inline yarn.lock:10353]: ?

File: src/actions/jobCategories.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        INCLUSION_IND: '0',
+      },
+      {
+        SKL_CODE: '2050',
+        SKL_DESC: 'INSPECTION (FUNCTIONAL)',
+        JCS_LAST_UPDT_USER_ID: 0,
+        JCS_LAST_UPDT_TMSMP_DT: null,
+        INCLUSION_IND: '0',
+      }],
+  };
```
Comment (scott062) [inline src/actions/jobCategories.js:68]: Lets remove dummy data and get it into the mock

File: src/Components/AdministratorPage/JobCategories/JobCategories.jsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+const JobCategories = () => {
+  const dispatch = useDispatch();
+  const jobCategories = useSelector(state => state.jobCategoriesAdminFetchData);
+  console.log('jobCategories: ', jobCategories);
+  const jobCategoriesResults = jobCategories?.data;
+  const jobCategorySkills = useSelector(state => state.jobCategoriesFetchSkills);
+  console.log('skills: ', jobCategorySkills);
+  // console.log('skills: ', jobCategorySkills?.results?.QRY_LSTSKILLS_REF);
+  const jobCategorySkillsResults = jobCategorySkills?.data;
+  // console.log('results: ', jobCategorySkillsResults);
```
Comment (M00staff) [inline src/Components/AdministratorPage/JobCategories/JobCategories.jsx:18]: so we're leaving all the commented code in - and then removing it in another PR?

Comment (pveland621) [reply src/Components/AdministratorPage/JobCategories/JobCategories.jsx]: just went ahead and cleaned it out

File: yarn.lock (hunk 1 above)
Comment (M00staff) [reply yarn.lock]: the next major version of this package throws a circle-ci error so i locked it into place, a few weeks ago... not sure why it's updating in this PR (prolly ran yarn install) - but that's the correct version.

**Review by M00staff — APPROVED** [review]

**Pull request merged**